k8s/pkg/apis/longhorn/v1beta2/recurringjob.go

```diff
@@ -33,12 +33,9 @@ type RecurringJobSpec struct {
 	// +optional
 	Cron string `json:"cron"`
 	// The retain count of the snapshot/backup.
-	// +kubebuilder:validation:Minimum=1
-	// +kubebuilder:validation:Maximum=50
```

Comment:
Why changes in recurring job and volume did not move to their validators?

Reply:
For node, the validation has already existed in https://github.com/longhorn/longhorn-manager/blob/master/webhook/resources/volume/validator.go#L58.

The recurringjob's upper bound is in https://github.com/longhorn/longhorn-manager/blob/master/webhook/resources/recurringjob/validator.go#L57, but the lower bound is missing.
So, I update the PR and add it explicitly.